applications/convection_diffusion_application/custom_elements/laplacian_element.cpp

```diff
@@ -76,28 +75,41 @@ void LaplacianElement::CalculateLocalSystem(MatrixType& rLeftHandSideMatrix, Vec
     //reading integration points and local gradients
     const GeometryType::IntegrationPointsArrayType& integration_points = GetGeometry().IntegrationPoints();
     const GeometryType::ShapeFunctionsGradientsType& DN_De = GetGeometry().ShapeFunctionsLocalGradients();
-    
+    const Matrix& N_gausspoints = GetGeometry().ShapeFunctionsValues();
+
     Element::GeometryType::JacobiansType J0;
     Matrix DN_DX(number_of_points,dim);
     Matrix InvJ0(dim,dim);
     Vector temp(number_of_points);
 
+    Vector heat_flux_local(number_of_points);
+    for(unsigned int node_element = 0; node_element<number_of_points; node_element++)
```

Comment:
for (std::size_t i_node=0; i_node<numpoints; ++i_node) {
...
}

Reply:
Thanks for the two hints!

Reply:
you did not correct either...